src/legacy/ledgerUtils.ts

```diff
@@ -492,3 +488,38 @@ export const signTxWithLedger = async (
     throw mapLedgerError(e)
   }
 }
+
+// not bumping react-native right now (couple to ledger)
+const BLUETOOTH_SCAN = 'android.permission.BLUETOOTH_SCAN'
+const BLUETOOTH_CONNECT = 'android.permission.BLUETOOTH_CONNECT'
+export const getLedgerPermissions = () => {
+  const permissions: Array<string> = [PermissionsAndroid.PERMISSIONS.ACCESS_FINE_LOCATION]
+  if (Platform.Version >= 31) {
+    permissions.push(BLUETOOTH_CONNECT)
+    permissions.push(BLUETOOTH_SCAN)
+  }
+  // hack
+  return permissions as unknown as Permission[]
```

Comment:
What I thought was to have it less hacky and use the `as` only when needed

```
const permissions: Array<Permission> = [PermissionsAndroid.PERMISSIONS.ACCESS_FINE_LOCATION]
  if (Platform.Version >= 31) {
    permissions.push(BLUETOOTH_CONNECT as Permission)
    permissions.push(BLUETOOTH_SCAN as Permission)
  }
  
  return permissions
```

Reply:
that's a good idea can you add it as suggestion so we can commit it?